### PR TITLE
DLDO - Placement fixes

### PR DESCRIPTION
**Pull request by lulu9312**
This PR fixes placement of blocks. Also added block of code to main ldo_gen.py which updates the ldo_custom_net.txt based on number of PT_UNIT_CELLS. The PLACE_DENSITY variable in config.mk will  also be updated based on number of PT_UNIT_CELLS.

### CODE REVIEW

File: openfasoc/common/platforms/sky130hvl/fastroute.tcl [hunk 1]
```diff
@@ -1,4 +1,5 @@
-set_global_routing_layer_adjustment $::env(MIN_ROUTING_LAYER)-$::env(MAX_ROUTING_LAYER) 0.5
+# reduces the routing resources of all routing layers by 30%
+set_global_routing_layer_adjustment * 0.3
```
Comment (msaligane) [inline openfasoc/common/platforms/sky130hvl/fastroute.tcl:2]: What is this added?

File: openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk [hunk 2]
```diff
@@ -14,17 +14,17 @@ export VERILOG_FILES 		= $(sort $(wildcard ./design/src/$(DESIGN_NICKNAME)/*.v))
 export CLK_PERIOD_IN_NS         = 1000
 export SDC_FILE    		= ./design/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
-export DIE_AREA   	 	= 0 0 280 380
-export CORE_AREA   		= 15 15 265 365
+export DIE_AREA   	 	= 0 0 340 340
```
Comment (msaligane) [inline openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk:17]: Have you been able to calculate this for each DLDO configuration? If so how?

**Comment by msaligane** [comment]
@lulu9312 This looks good.
We need to add test benches and regression tests for blocks such as the Vref and comparator, DLDO. I am leaning towards FoM usually published but we can add simple tests as a first step.
These should be added as part of the advanced we are discussing with @saicharan0112

File: openfasoc/common/platforms/sky130hvl/fastroute.tcl (hunk 1 above)
Comment (lulu9312) [reply openfasoc/common/platforms/sky130hvl/fastroute.tcl:2]: While using earlier command the adjustment setting was not changed in the flow. The updated command will change the setting for metal1 to metal5 .

File: openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk (hunk 2 above)
Comment (lulu9312) [reply openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk:17]: As of the the area is fixed. I tested the flow with various number of switches. The only parameter that should be changed is PLACE_DENSITY for varying number of switches. And that has been implemented in the flow.

File: openfasoc/common/platforms/sky130hvl/fastroute.tcl (hunk 1 above)
Comment (msaligane) [reply openfasoc/common/platforms/sky130hvl/fastroute.tcl:2]: Why are you reducing routing resources by 30% 
`set_global_routing_layer_adjustment * 0.3`

File: openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk (hunk 2 above)
Comment (msaligane) [reply openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk:17]: This should be automated based on the instances count out of synthesis and modeling. Can you please add it as a Github issue and assign it to yourself?

**Review by msaligane — APPROVED** [review]

File: openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk (hunk 2 above)
Comment (lulu9312) [reply openfasoc/generators/ldo-gen/flow/design/sky130hvl/ldo/config.mk:17]: Yes.

File: openfasoc/common/platforms/sky130hvl/fastroute.tcl (hunk 1 above)
Comment (lulu9312) [reply openfasoc/common/platforms/sky130hvl/fastroute.tcl:2]: This is a default setting that is been used by OpenROAD-flow-scripts.

**Pull request merged**